src/ListItem.tsx

```diff
@@ -8,27 +19,99 @@ export interface person {
 }
 
 export function ListPage(props: { list: person[] }): JSX.Element {
+  const [upvoteClicked, changeUpvote] = useState(false);
+  const [downvoteClicked, changeDownvote] = useState(false);
+
   return (
-    // Looks of Page
     <>
-      <div>
-        <h1 className="title">Rate My Professor</h1>
-      </div>
-
       {/* does the indexing work for you */}
       {props.list.map((person: person, index: number) => (
-        <div
-          style={{
-            backgroundColor: 'dark grey',
-            marginTop: '10px',
-          }}
-          key={index}
-        >
-          <p className="professorName">{person.professorName}</p>
-          <p className="overallRating">{person.overallRating}</p>
-          <p className="difficulty">{person.difficulty}</p>
-          <p className="reviewCount">{person.reviewCount}</p>
-        </div>
+        <Accordion key={index}>
+          <AccordionSummary
+            expandIcon={<ExpandMoreIcon sx={{ fontSize: '2.5rem' }} />}
+            aria-controls="panel1a-content"
+            id="panel1a-header"
+          >
+            <Typography>
+              {person.professorName}{' '}
+              <b>{person.overallRating} / 5 Overall Rating</b>
```

Comment:
Preferably have Overall Rating be its own line, and just use "Rating" instead of both words for UX, i.e.
- Rating {person.overallRating} / 5

There's also a random space between the ratings, it's a little strange but not that big of a deal
![image](https://user-images.githubusercontent.com/24869207/211094571-85a09d64-75a6-419b-b165-dc71fb72e963.png)